Jenkinsfile

```diff
@@ -482,6 +482,17 @@ try {
         }
       }
     },
+    'Scala: GPU': {
+      node('mxnetlinux-gpu') {
+        ws('workspace/ut-scala-gpu') {
+          init_git()
+          unpack_lib('gpu')
+          timeout(time: max_time, unit: 'MINUTES') {
+            sh "ci/build.py --build --platform ubuntu_gpu /work/runtime_functions.sh unittest_ubuntu_gpu_scala"
```

Comment:
please add --nvidia-docker in order to make sure this gets executed in a GPU container. It's interesting CI passed although no GPUs were present. Do you have an idea why this could be the case?

Reply:
CI didnt pass for Scala-gpu test

Reply:
Oops, seems like I mixed something up. My bad, thanks for pointing it out @Roshrini :)